### PR TITLE
bumping pypi to v1.0.2

### PR DESCRIPTION
**Pull request by gabrielilharco**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ## History
 
+### 1.0.2
+
+* Added support for multilingual eval
+* Added support for linear probing
```
Comment (mitchellnw) [inline HISTORY.rst:6]: there was already linear probing, just not the ability to run exps with python script

**Comment by gabrielilharco** [comment]
@rom1504 let me know if there are any changes/additions I should make on the history

**Comment by rom1504** [comment]
Can you make it 1.1.0 ?

Adding features should be a minor version as per https://semver.org/

**Comment by gabrielilharco** [comment]
Sure thing, it's done

**Pull request merged**